src/rubrix/metrics/__init__.py

```diff
@@ -0,0 +1,33 @@
+from .commons import records_status, text_length
+from .text_classification import f1, f1_multilabel
+from .token_classification import (
+    entity_capitalness,
+    entity_consistency,
+    entity_density,
+    entity_labels,
+)
+from .token_classification import f1 as ner_f1
+from .token_classification import (
+    mention_length,
+    token_capitalness,
+    token_frequency,
+    token_length,
+    tokens_length,
+)
+
+__ALL__ = [
```

Comment:
Can we "reuse" Python's [__all__](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) for this?

Reply:
sure.